ci/tests/plugin-compiler/testplugin/go.mod

```diff
@@ -3,28 +3,10 @@ module github.com/TykTechnologies/tyk/ci/tests/plugin-compiler/testplugin
 go 1.16
 
 require (
-	github.com/HdrHistogram/hdrhistogram-go v1.1.0 // indirect
 	github.com/Masterminds/sprig/v3 v3.2.2
-	github.com/TykTechnologies/murmur3 v0.0.0-20190927072507-ba59b2844ad7 // indirect
-	github.com/TykTechnologies/tyk v1.9.2-0.20210625184536-6b5eac3429dd
-	github.com/buger/jsonparser v1.1.1 // indirect
-	github.com/franela/goblin v0.0.0-20210519012713-85d372ac71e2 // indirect
-	github.com/google/uuid v1.2.0 // indirect
-	github.com/huandu/xstrings v1.3.2 // indirect
-	github.com/imdario/mergo v0.3.12 // indirect
-	github.com/kr/pretty v0.1.0
-	github.com/mattn/go-colorable v0.1.8 // indirect
-	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d // indirect
-	github.com/mitchellh/copystructure v1.2.0 // indirect
-	github.com/onsi/ginkgo v1.16.4 // indirect
-	github.com/onsi/gomega v1.13.0 // indirect
-	github.com/opentracing/opentracing-go v1.2.0 // indirect
-	github.com/sirupsen/logrus v1.8.1 // indirect
-	github.com/uber/jaeger-client-go v2.29.1+incompatible // indirect
-	github.com/uber/jaeger-lib v2.4.1+incompatible // indirect
-	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
-	go.uber.org/atomic v1.8.0 // indirect
-	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e // indirect
+	github.com/TykTechnologies/tyk v1.9.2-0.20230606201232-e599d84bdfd1
```

Comment:
do we need to update this to the latest 5.1 commit?

Reply:
yes, good catch, opening a second PR

Reply:
Actually, no, we don't need to right now, it's plugin compiler (i asumed it was the main /go.mod). Plugin compiler QA tests are something different and this test area is broken and RFC/issues in place around it. We can leave it broken until we figure out how to address it.